jsapp/js/router/accessDenied.tsx

```diff
@@ -17,14 +17,48 @@ export interface AccessDeniedProps {
 }
 const AccessDenied = (props: AccessDeniedProps) => {
   let messageText;
+  let headerText;
+  let bodyText;
+  let errorNumber;
+  const loggedIn = t(
+    `Please try logging in using the header button or [contact the support team] if you think it's an error.`
+  );
+  const loggedOut = t(
+    `Please [contact the support team] if you think it's an error.`
+  );
```

Comment:
The text of these two messages should be switched. When I go to the error page while logged in, it tells me to log in:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/538d66f7-f24f-485a-981c-64efdcc71413)

And it doesn't tell me to log in if I'm logged out:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/939bb65d-d425-42b7-8973-092fcfe4da96)